internal/comment/comment_test.go

```diff
@@ -0,0 +1,23 @@
+package comment_test
+
+import (
+	"testing"
+	"github.com/stretchr/testify/assert"
+	"grab/internal/comment"
+	"grab/internal/comment/mocks"
+	"encoding/json"
+)
+
+func TestGetComments(t *testing.T) {
+	mockCommentRep := &mocks.CommentsWorker{}
+	var getComments []comment.Comment
+	var expectedRes []comment.Comment
+
+	mockCommentRep.On("GetComments").Return(getComments, nil)
```

Comment:
This line always returns an empty array because your `getComments` variable is an empty array. You should assign some values (same value with `trueData`) for it then your test will pass.